src/Sulu/Bundle/ContentBundle/Resources/public/js/components/content/column/main.js

```diff
@@ -45,18 +47,31 @@ define([
 
         ACTION_ICON_VIEW = 'fa-eye',
 
+        ACTION_ICON_INFO = 'fa-info',
+
+
+        brokenPageDialogTranslations = {
```

Comment:
because its not used completelly in this file but your right i should use it anyway